.golangci.yml

```diff
@@ -9,23 +9,23 @@ run:
 linters:
   disable-all: true
   enable:
-    - prealloc
+    - depguard
+    - exportloopref
+    - gocritic
+    - goimports
+    - gosimple
     - ineffassign
-    - goerr113
     - misspell
-    - errcheck
-    - gosimple
+    - prealloc
     - staticcheck
-    - gosec
-    - gocritic
-    - unparam
-    - deadcode
```

Comment:
Delete deadcode because this linter is deprecated.
cf. https://github.com/golangci/golangci-lint/issues/1841